tests/layer_tests/pytorch_tests/test_logaddexp.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestLogAddExp(PytorchLayerTest):
+    def _prepare_input(self, input1, input2, dtype="float32"):
+        """Prepare inputs for logaddexp testing"""
+        return (np.array(input1).astype(dtype), np.array(input2).astype(dtype))
+
+    def create_model(self, dtype=None):
+        import torch
+
+        dtype_map = {
+            "float32": torch.float32,
+            "float64": torch.float64,
+        }
+
+        class LogAddExpModel(torch.nn.Module):
+            def __init__(self, dtype=None):
+                super(LogAddExpModel, self).__init__()
+                self.dtype = dtype_map.get(dtype) if dtype else None
+
+            def forward(self, x, y):
+                if self.dtype:
+                    x = x.to(self.dtype)
+                    y = y.to(self.dtype)
+                return torch.logaddexp(x, y)
+
+        model_class = LogAddExpModel(dtype)
+        ref_net = None
+
+        return model_class, ref_net, "aten::logaddexp"
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize(
```

Comment:
please also add `@pytest.mark.precommit_torch_export`. Take a look how other operations are enabled for torch.export case.
In order to test it locally, set env var `PYTORCH_TRACING_MODE=EXPORT`: https://github.com/openvinotoolkit/openvino/blob/master/.github/workflows/job_pytorch_layer_tests.yml#L132